macros/models/_sector/cex/cex_flows.sql

```diff
@@ -29,8 +21,10 @@ SELECT t.blockchain
 , t.tx_hash
 , t.evt_index
 , t.unique_key
-FROM {{ ref('tokens_ethereum_transfers')}} t
-INNER JOIN {{ ref('cex_ethereum_addresses')}} a ON a.address IN (t."from", t.to)
+FROM {{transfers}} t
+INNER JOIN {{addresses}} a ON a.address IN (t."from", t.to)
```

Comment:
can you run a quick test on performance to see how runtime looks when you remove `IN` and do `(a.address = t."from" or a.address = t.to)`?

Reply:
old run for reference:
https://github.com/duneanalytics/spellbook/actions/runs/8178326082/job/22361994788

can compare when new is done

Reply:
okay, no real change. feel free to keep as-is or revert to your preference

Reply:
it's good as is, feel free to merge